orderer/consensus/etcdraft/util_test.go

```diff
@@ -391,30 +370,46 @@ func TestVerifyConfigMetadata(t *testing.T) {
 		})
 	}
 
-	// test use case when consenter has expired certificates
-	tlsCaCertBytes, err := ioutil.ReadFile(filepath.Join(ca1Dir, "ca.pem"))
-	require.Nil(t, err)
-	tlsCaCert, err := parseCertificateFromBytes(tlsCaCertBytes)
-	require.Nil(t, err)
+	t.Run("ExpiredCertificate", func(t *testing.T) {
+		clientPair, err := tlsCA.NewClientCertKeyPair()
+		require.NoError(t, err, "failed to create client key pair")
 
-	tlsClientCert, err := ioutil.ReadFile(filepath.Join(ca1Dir, "client3.pem"))
-	require.Nil(t, err)
+		clientCert, err := parseCertificateFromBytes(clientPair.Cert)
```

Comment:
Not a big deal, but there is already a `TLSCert *x509.Certificate` hanging off the `clientPair`, so we should just be able to use that?

Reply:
I'll make use of that.